barcodes_generator_abstract/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2014-Today GRAP (http://www.grap.coop)
+# Copyright (C) 2016-Today La Louve (http://www.lalouve.net)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Generate Barcodes (Abstract)',
+    'summary': 'Generate Barcodes for Any Models',
+    'version': '9.0.1.0.0',
+    'category': 'Tools',
+    'author':
+        'GRAP,'
+        'La Louve,'
+        'Odoo Community Association (OCA)',
+    'website': 'http://www.odoo-community.org',
+    'license': 'AGPL-3',
+    'depends': [
+        'barcodes',
+    ],
+    'data': [
```

Comment:
Well in fact, it's 'barcode', simply, but yes ! Thanks.
https://github.com/OCA/stock-logistics-barcode/pull/49/commits/5e78cabb5125aa2b6cf8788681d4bd1d6cc151c7#diff-3f9fcad59a91c0ecef85bbd7cde75c9c